plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationProviderImpl.kt

```diff
@@ -28,9 +29,11 @@ internal class LocationProviderImpl(context: Context) : LocationProvider, Locati
 
   @SuppressLint("MissingPermission")
   private fun requestLocationUpdates() {
-    locationEngine.requestLocationUpdates(
-      locationEngineRequest, this, Looper.getMainLooper()
-    )
+    if (PermissionsManager.areLocationPermissionsGranted(context)) {
```

Comment:
shall we add some error log when the permission is not granted at this point of time?

Reply:
Yes, will do.